pkg/features/features.go

```diff
@@ -54,7 +54,10 @@ const (
 )
 
 func init() {
-	utilfeature.DefaultFeatureGate.Add(defaultFederationFeatureGates)
+	if err := utilfeature.DefaultFeatureGate.Add(defaultFederationFeatureGates); err != nil {
+		// TODO:
```

Comment:
What should I do when `Add` return error

Reply:
This is a pretty low value fix, I suggest leaving it for now.

Reply:
I realize I was confused about what you were asking.  I think `glog.Fatalf` is likely the right answer.